MatrixSDK/VoIP/CallStack/Jingle/MXJingleCallAudioSessionConfigurator.h

```diff
@@ -0,0 +1,53 @@
+/*
+ Copyright 2017 Vector Creations Ltd
+ 
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+ 
+ http://www.apache.org/licenses/LICENSE-2.0
+ 
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+#import <Foundation/Foundation.h>
+
+#import "MXSDKOptions.h"
+
+#ifdef MX_CALL_STACK_JINGLE
+
+@class AVAudioSession;
+
+/**
+ This class helps to provide better integration level with CallKit for WebRTC
+ */
+@interface MXJingleCallAudioSessionConfigurator : NSObject
```

Comment:
I think it worths to abstract that into a MXCallAudioSessionConfigurator protocol. MXJingleCallAudioSessionConfigurator would be the implementation for Jingle.

VoIP stacks have always their own specific things. I prefer to abstract things as much as possible.